test/testdata/rbs/signatures_generics.rb

```diff
@@ -0,0 +1,154 @@
+# typed: strict
+# enable-experimental-rbs-signatures: true
+# enable-experimental-rbs-assertions: true
```

Comment:
Is it possible to use the new generics syntax with only one of these assertions? Should we have a test for that?

Reply:
Yes but you can't instantiate:

```rb
#: [U]
class G; end

g = G.new #: G[Integer]
T.reveal_type(g) # error: Revealed type: `G[T.untyped]`
```

I added a test for that.

We also plan to merge these two options into a `--rbs` in the future, it was mostly useful while we were building the features.